cmd/limactl/show_ssh.go

```diff
@@ -42,8 +45,13 @@ const showSSHExample = `
 
 func newShowSSHCommand() *cobra.Command {
 	var shellCmd = &cobra.Command{
-		Use:               "show-ssh [flags] INSTANCE",
-		Short:             "Show the ssh command line",
+		Use:   "show-ssh [flags] INSTANCE",
+		Short: "Show the ssh command line (DEPRECATED; use `ssh -F` instead)",
+		Long: `Show the ssh command line (DEPRECATED)
+
+WARNING: 'limactl show-ssh' is deprecated.
+Instead, use 'ssh -F ~/.lima/default/ssh.config lima-default' .
```

Comment:
Should probably not hardcode LIMA_HOME here

Reply:
Fixed